### DISCUSSION TITLE
Initiate Gitea PullRequest manually

### DISCUSSION THREAD

**Discussion by buhtz** [WeblateOrg/weblate#8535] [Q&A]
I disabled _Push on commit_ because every translation caused a PR on my upstream repo. That is a bit annoying to me and pollutes my branches.

But I would like to trigger the creation of a PR in the managment/maintaince section of weblate.

There is a button "Push" which just gives me the error that it doesn't work. ;) The history tells me that "Fork creation failed: ".

I don't want to Push, just PR. How can I do this?

**Comment by nijel** [comment]
"Push" is the correct button for this – it pushes changes to a fork or upstream repo, and creates a pull request.

The error is indeed weird - the create fork API doesn't return forked repository, but also doesn't provide an error message: https://try.gitea.io/api/swagger#/repository/createFork
